scrapy_zyte_smartproxy/middleware.py

```diff
@@ -35,6 +36,15 @@ class ZyteSmartProxyMiddleware(object):
     max_auth_retry_times = 10
     enabled_for_domain = {}
     apikey = ""
+    zyte_api_to_spm_translations = {
+        b"zyte-client": b"x-crawlera-client",
+        b"zyte-device": b"x-crawlera-profile",
+        b"zyte-geolocation": b"x-crawlera-region",
+        b"zyte-jobid": b"x-crawlera-jobid",
+        b"zyte-no-bancheck": b"x-crawlera-no-bancheck",
```

Comment:
Even though we don't support `zyte-no-bancheck` we can let this be since we will be adding support for it soon